stdlib/public/Platform/Platform.swift

```diff
@@ -187,6 +209,7 @@ public func open(
   return _swift_Platform_open(path, oflag, 0)
 }
 
+#if !os(Windows) || CYGWIN
 public func open(
   _ path: UnsafePointer<CChar>,
   _ oflag: Int32,
```

Comment:
Ugh, I see what you mean.  This was supposed to have an alternate implementation with the signature change.  Ill fix this in the upload that Im about to do.